config/initializers/okcomputer.rb

```diff
@@ -33,6 +33,22 @@ def table_check(klass)
   end
 end
 
+class OcrQueueDepthCheck < OkComputer::Check
+  MAX_OCR_WAITING = 20 # alert if we have more than this number of objects waiting in ocrWF:ocr-create
+
+  def check
+    num_ocr_waiting = SearchService.query('wf_wps_ssim:ocrWF:ocr-create:waiting', fl: 'id')['response']['numFound']
```

Comment:
I guess there's not a way to get this from Workflow Service?

Reply:
we could, but i think the local solr query is faster than making an API call to an external service

Reply:
Ok. I don't think speed is an issue here, as much as using the microservice architecture that has been built? But that's your call.

I can't decide whether Solr as a scratch space for everything is magic I should be grateful for or an anti-pattern to be avoided at all costs. I kind of lean towards the latter.

Reply:
I'm with @edsu on this one.  This should be a db query in WFS.

Reply:
okcomputer check via db query in WFS makes sense ... i'll do that instead